tests/storage/stress/thinp.py

```diff
@@ -5,52 +5,73 @@
 
```

Comment:
> Add command line arguments to there is no need to edit the script.

"to" -> "so"

> Add correction time for throttling, fixing considering the time to  sleep and get the next timestamp in the next sleep.

either "fixing" or "considering"

Reply:
Thanks, fixed.